services/score/queueSenderForRedis.js

```diff
@@ -13,6 +13,14 @@ const removeActivityQueue = new Bull(QUEUE_REMOVE_ACTIVITY, connectRedis, {
   }
 });
 
+const unFollowFeedProcessQueue = new Bull(QUEUE_UNFOLLOW_FEED_PROCESS, connectRedis, {
+  redis: {
+    ...bullConfig,
+    maxRetriesPerRequest: 100,
+    connectTimeout: 30000
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The configuration for `unFollowFeedProcessQueue` is hardcoded. It would be better to move these values to a configuration file or environment variables, which would make it easier to manage and change them without modifying the code.
```diff
-  const unFollowFeedProcessQueue = new Bull(QUEUE_UNFOLLOW_FEED_PROCESS, connectRedis, {
-    redis: {
-      ...bullConfig,
-      maxRetriesPerRequest: 100,
-      connectTimeout: 30000
-    }
-  });
+  const unFollowFeedProcessQueue = new Bull(QUEUE_UNFOLLOW_FEED_PROCESS, connectRedis, bullConfig);
```
In this case, you should add `maxRetriesPerRequest` and `connectTimeout` to your `bullConfig`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->